packages/cli/src/index.ts

```diff
@@ -86,4 +85,5 @@ yargsCli(process.argv.slice(2))
     },
   })
   .help()
-  .wrap(72).argv;
```

Comment:
`.argv` was never correct here. I the `parse` call is redundant in this case, but I added it anyway.